scripts/create-package.js

```diff
@@ -67,7 +67,6 @@ const steps = [
   { template: 'TsLint', output: 'tslint.json' },
   { template: 'WebpackConfig', output: 'webpack.config.js' },
   { template: 'WebpackServeConfig', output: 'webpack.serve.config.js' },
-  { template: 'VsCodeLaunch', output: path.join('.vscode', 'launch.json') },
```

Comment:
@khmakoto I realized that my comment about the not very useful launch.json on your recent PR applies in general, so I'm removing it from the template and removing all the ones generated from the template.